site/src/pages/CreateUserPage/CreateUserForm.tsx

```diff
@@ -124,6 +129,17 @@ export const CreateUserForm: FC<
             fullWidth
             label={Language.usernameLabel}
           />
+          <TextField
+            {...getFieldHelpers("name")}
+            onBlur={(e) => {
+              e.target.value = e.target.value.trim();
+              form.handleChange(e);
+            }}
```

Comment:
review: this is already done in the profiile edit view, keeping here for consistency.
Backend already trims, this is just for safety+redundancy.

Reply:
Honestly, I would remove `onBlur` from both places since it is unnecessary.